### PR TITLE
[cCc] sweep

### PR DESCRIPTION
**Pull request by r-ferrier**
addresses #109 (work in this PR ready to be checked and merged, but should not close this issue - we need to check everything again as some copy has still not been updated)

## Description

- Updates all approved copy from the google sheet
- Removes all test markdown guide and story content
- Moves all links and non translatable content from keys into code
- Removes keys and display of content from keys that are no longer required

@geeksforsocialchange/developers

### CODE REVIEW

File: src/Theme/FooterTemplate.elm [hunk 1]
```diff
@@ -54,7 +54,7 @@ translatedLogoPath language fileName =
             "/images/" ++ fileName ++ "-cy.svg"
 
 
-navigationColumn : { title : Key, links : List { text : Key, href : Key } } -> Language -> Route -> Html msg
+navigationColumn : { title : Key, links : List { text : Key, href : String } } -> Language -> Route -> Html msg
```
Comment (ivan-kocienski-gfsc) [inline src/Theme/FooterTemplate.elm:57]: I am worried that by using plain strings we may introduce un-catchable errors with the links. 

This is okay for now but maybe there could be some kind of formal "link" database we can key into so at least all the strings are in one place.

File: src/I18n/Cy.elm [hunk 2]
```diff
@@ -174,10 +160,10 @@ cyStrings key =
             "/images/icons/NextDoorNature_icons-text.svg"
 
         GuideButtonText ->
-            "[cCc]"
+            "Cliciwch yma am fersiwn argraffadwy o'r canllaw hwn"
 
         GuideParagraphText ->
-            "[cCc]"
+            ""
```
Comment (ivan-kocienski-gfsc) [inline src/I18n/Cy.elm:166]: should this be empty?

File: src/Theme/FooterTemplate.elm (hunk 1 above)
Comment (r-ferrier) [reply src/Theme/FooterTemplate.elm:57]: I agree with what you're saying, but the links have to be hardcoded somewhere and I'm not sure that adding another file for this necessarily stops this from being a problem - it just shifts the problem elsewhere. I did try specifying them all as separate functions but it felt very OTT, so this navigation column function is in effect now just a type of data structure. Definitely something to consider in the future.

Comment (ivan-kocienski-gfsc) [reply src/Theme/FooterTemplate.elm:57]: i think it's more- if you're doing something potentially hacky then at least hide it in one place with a nice interface?

but yeah- not an issue at this time

**Review by ivan-kocienski-gfsc — CHANGES_REQUESTED** [review]
Just one thing really

File: src/I18n/Cy.elm (hunk 2 above)
Comment (ivan-kocienski-gfsc) [reply src/I18n/Cy.elm:166]: this is accepted copy.

**Review by ivan-kocienski-gfsc — APPROVED** [review]
Seems okay.

**Pull request merged**